src/react-components/avatar-editor.js

```diff
@@ -487,16 +516,25 @@ export default class AvatarEditor extends Component {
               <AvatarPreview
                 className="preview"
                 avatarGltfUrl={this.state.previewGltfUrl}
+                onGltfLoaded={this.handleGltfLoaded}
```

Comment:
I recognize its pretty strange to have the AvatarPreview be part of this flow, but it is already loading the GLTF and decoupling that part of it would be kinda messy, so it doesn't seem worth it for this one-off feature.